libs/blocks/preflight/panels/general.js

```diff
@@ -72,8 +72,12 @@ function findLinks(selector) {
   return [...document.body.querySelectorAll(selector)]
     .reduce((links, el) => {
       const url = getUrl(el);
-      if (!hrefs.includes(url.href)) {
-        hrefs.push(url.href);
+      const baseUrl = url.origin + url.pathname;
+      if (url.pathname.includes('/tools/caas')) {
```

Comment:
We might have additional paths to exclude in the future. How about storing such paths somewhere and checking against them here?